src/plugins/vis_type_timeseries/kibana.json

```diff
@@ -4,7 +4,7 @@
   "kibanaVersion": "kibana",
   "server": true,
   "ui": true,
-  "requiredPlugins": ["charts", "data", "expressions", "visualizations"],
+  "requiredPlugins": ["charts", "data", "expressions", "visualizations", "visualize"],
```

Comment:
`visualize` should be optional for the case if it is disabled, the same as for `vis_default_editor` plugin

Reply:
with these changes `TSVB` registers its editor via the `visualize` plugin. If the latter is blocked, it will not be possible to register the editor for `TSVB`. It sounds strange to me ...